Frontend/src/app/components/lean-lab/lean-lab.component.scss

```
@@ -71,103 +103,132 @@
-    column-gap: 0.75rem;
-    row-gap: 0.25rem;
+    column-gap: var(--space-3);
+    row-gap: var(--space-1);
     margin: 0;
-    font-size: 0.85rem;
+    font-size: var(--fs-base);
   }
 
-  dt { font-weight: 600; }
+  dt { font-weight: var(--fw-semi); }
   dd { margin: 0; word-break: break-word; }
```

Stylelint (17.11.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)